library/std/src/fs.rs

```diff
@@ -1365,6 +1365,30 @@ impl FileTimes {
 impl Permissions {
     /// Returns `true` if these permissions describe a readonly (unwritable) file.
     ///
+    /// Warning: This function does not take ACLs or Unix group membership into
+    /// account.
```

Comment:
How about making it a header to get more attention?

```suggestion
    /// # Note
    ///
    /// This function does not take ACLs or Unix group membership into
    /// account.
```

Reply:
Good idea - I did look into how to make it a nice big yellow warning but it seems like support for that is still a WIP. See #73935 & #79710. This is a decent enough substitute for now though.